.github/workflows/mirror-calico.yml

```
@@ -0,0 +1,57 @@
+name: Sync GHCR Calico images with Docker Hub
+on:
+  schedule:
+    # run every 12h
+    - cron:  '0 */12 * * *'
+  workflow_dispatch:
+
+jobs:
+  get-docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to GitHub Container Registry (ghcr)
+        run: echo ${{ secrets.GHCR_PASSWORD }} | docker login ghcr.io -u ${{ secrets.GHCR_USERNAME }} --password-stdin
+      - name: Fetch latest Calico release
+        id: fetch-latest-release
+        run: |
+          set -ex
+
+          git clone https://github.com/projectcalico/calico calico
+          # get latest version
+          version=$(git -C $_ tag | sort -V | tail -1)
+
+          # list of images to mirror from Docker Hub
+          images=(
+            calico/typha
+            calico/pod2daemon-flexvol
+            calico/cni
+            calico/node
+            calico/kube-controllers
+          )
+
+          # we want both arch for running tests
+          platforms=(
+            amd64
+            arm64
+          )
+
+          # we iterate over the images we want to mirror
+          for image in "${images[@]}"; do
+            # tags will hold the images the mirrored images
```

Not sure about this one.

```suggestion
            # tags will hold the names of the mirrored images
```

---

It's actually:
```suggestion
            # tags will hold the mirrored images
```

---

Sounds good.